seq2seq/prediction_output.py

```diff
@@ -0,0 +1,188 @@
+# Set up logging
+import sys
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+    handlers=[logging.StreamHandler(sys.stdout)],
+    level=logging.WARNING,
+)
+logger = logging.getLogger(__name__)
+
+from typing import Optional
+from dataclasses import dataclass, field
+import os
+import json
+from contextlib import nullcontext
+from alive_progress import alive_bar
+from transformers.hf_argparser import HfArgumentParser
+from transformers.models.auto import AutoConfig, AutoTokenizer, AutoModelForSeq2SeqLM
+from seq2seq.utils.pipeline import ConversationalText2SQLGenerationPipeline, Text2SQLGenerationPipeline, Text2SQLInput, ConversationalText2SQLInput
+from seq2seq.utils.picard_model_wrapper import PicardArguments, PicardLauncher, with_picard
+from seq2seq.utils.dataset import DataTrainingArguments
+
+
+@dataclass
+class PredictionOutput:
```

Comment:
```suggestion
class PredictionOutputArguments:
```
Otherwise, it sounds like this is a data class for the outputs themselves.